source/MaterialXTest/XmlIo.cpp

```diff
@@ -40,6 +41,34 @@ TEST_CASE("Load content", "[xmlio]")
         libs.push_back(lib);
     }
 
+    // Check that there is one implementation per nodedef.
+    mx::DocumentPtr implCheckDocument = mx::createDocument();
+    for (mx::DocumentPtr lib : libs)
+    {
+        implCheckDocument->importLibrary(lib);
+    }
+    const std::string target;
+    const std::string language("osl");
+    std::vector<mx::NodeDefPtr> definitions = implCheckDocument->getNodeDefs();
+    std::set<std::string> definitionsFound;
+    for (mx::NodeDefPtr definition : definitions)
+    {
+        // Ignore untyped nodedefs as they require no definition
```

Comment:
Following the advice above, I would say that these nodedefs "require no implementation", to steer clear of potential ambiguity in the term "definition".